plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import interpolateComponents from '@automattic/interpolate-components';
+import { Button, Card, CardBody, CardMedia } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { Link, Pill } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './incentive-banner.scss';
+import IncentiveIcon from './incentive-icon.svg';
```

Comment:
I know we have a lot of code that use SVG in JS files 😅, but it seems that we should only use it when we need to set its props dynamically (pe4Hk0-1d6-p2#comment-1459).

Reply:
Interesting, I wasn't aware of that so thanks for sharing! Sorry if I missed it in the PR link, but is there an example of the correct way to use SVG-in-JS somewhere in the Core codebase already?

Reply:
Yeah, we have a lot of examples of that. It's actually not SVG-in-JS, but normal asset files.

You can put the SVG file in the `plugins/woocommerce/assets/images` folder and then use it like this:

https://github.com/woocommerce/woocommerce/blob/15a3ca848d0f28c9dffc26a2374734db129003e1/plugins/woocommerce/client/admin/client/task-lists/setup-task-list/components/task-headers/customize-store.tsx#L35-L38